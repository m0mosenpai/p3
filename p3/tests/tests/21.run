../solution/wsh <tests/21.wsh
