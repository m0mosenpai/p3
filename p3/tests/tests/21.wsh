echo hello
echo hello
echo hello
echo hello
echo hello
history
