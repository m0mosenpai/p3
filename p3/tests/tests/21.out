wsh> hello
wsh> hello
wsh> hello
wsh> hello
wsh> hello
wsh> 1) echo hello
wsh> 